routes/html-routes.js

```diff
@@ -1,10 +1,53 @@
 // Dependencies
 // =============================================================
-var path = require("path");
+var express = require('express');
+var router = express.Router();
 
 
 // Routes
 // =============================================================
-module.exports = function(app) {
-    
-}
\ No newline at end of file
+router.get("/", function (req, res) {
+    console.log(req.method, "request completed.");
+
+
+    res.send("Log in page goes here.")
+    return;
```

Comment:
What is this return?

Reply:
The return is just to keep the res.render function to run since we don't have a handlebar view to put into that page just yet.